mindmeld/markup.py

```diff
@@ -127,13 +127,17 @@ def read_query_file(file_path):
     Yields:
         str: query text for each line
     """
-    with codecs.open(file_path, encoding='utf-8') as queries_file:
-        for line in queries_file:
-            line = line.strip()
-            # only create query if line is not empty string
-            query_text = line.split('\t')[0].strip()
-            if query_text:
-                yield query_text
+    try:
+        with codecs.open(file_path, encoding='utf-8') as queries_file:
+            for line in queries_file:
+                line = line.strip()
+                # only create query if line is not empty string
+                query_text = line.split('\t')[0].strip()
+                if query_text:
+                    yield query_text
+    except IOError:
```

Comment:
just wondering, if the file can't be opened or parsed, are we ok with the application to keep running?  It seems like this should be a service ending error, let me know if I'm wrong.

Reply:
My idea is that we still want to keep the application running even if one file in the training data set is corrupted.

Reply:
my main concern is the error might be missed or ignored by the deployer because the error is  not escalated enough because the app is still running and will give a false sense that the app is running correctly.  Don't think the issue is big enough to stop merge though.